R/tufte_book.R

```diff
@@ -0,0 +1,86 @@
+
+#' Tufte book format (PDF)
+#' 
+#' Template for creating a book according to the style of 
+#' Edward R. Tufte and Richard Feynman. 
+#' 
+#' @inheritParams pdf_document
+#' 
+#' @export
+tufte_book <- function(fig_width = 4,
+                          fig_height = 2.5,
+                          fig_crop = TRUE,
+                          highlight = "default",
+                          keep_tex = FALSE,
+                          includes = NULL,
+                          pandoc_args = NULL,
+                          toc = TRUE,
+                          toc_depth = 3,
+                          latex_engine = "xelatex",
+                          number_sections = TRUE,
+                          ...
```

Comment:
Any argument we think is relevant / supported should be included in the signature (for documentation to end users of what is expected to work).